### ISSUE TITLE
Replace border style in components to meet AA

### ISSUE THREAD

**Issue by panchovm** [WordPress/openverse-frontend#582]
## Description
Many components have a border style using `dark-charcoal-20` for resting and hover states. This color does not meet the AA contrast ratio for Graphics.

## Expectation
To meet the contrast ratio required, we need to use a darker version of shade palette. That is why I tested the available options and ended up replacing the `dark-charcoal-20` with `dark-charcoal-50`. I updated the components in the [Design Library](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library) and here is a list with all elements modified.

#### Navigation
* [Search input](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=558%3A2278) (#584)
* [Content switcher](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=558%3A2277)
* [Filters](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=558%3A2276)
* [Filters value](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=558%3A2275)
* [Internal menu](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=558%3A2274)

#### Button
* [Search button](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=699%3A3200) (#584)
* [Content link](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=558%3A2284)

### Views
* [Header. Desktop](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=751%3A2802)
* [Header. Mobile](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=751%3A2808)

**Comment by panchovm** [comment]
This ticket is no longer valid since the problem detected was incorrect. The [a11y requirement for non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) allows the use of a lighter shade as border for the components listed.
